dispatches/models/nuclear_case/unit_models/hydrogen_turbine_unit.py

```diff
@@ -132,13 +132,11 @@ def build(self):
         TransformationFactory("network.expand_arcs").apply_to(self)
 
     def initialize(self, state_args=None,
-                   solver=None, outlvl=idaeslog.NOTSET):
+                   solver=None, optarg=None, outlvl=idaeslog.NOTSET):
 
         init_log = idaeslog.getInitLogger(self.name, outlvl, tag="unit")
 
         if solver is None:
```

Comment:
Why the `if solver is None:`?

Also, no passing the `solver` and `optarg` arguments to `get_solver`.